.github/workflows/review.yml

```diff
@@ -34,6 +34,7 @@ jobs:
           pytest.ini
           *.yapf
           COPYING*
+          tests/*.md
```

Comment:
we already have `*.md` so itshould be covered I think.
If not, please look for option to skip all md files by one `files_ignore` entry, maybe:
```*/*.md```